system/modules/core/templates/jquery/j_mediaelement.html5

```diff
@@ -9,7 +9,12 @@ $GLOBALS['TL_CSS'][] = 'assets/jquery/mediaelement/'. $GLOBALS['TL_ASSETS']['MED
 <script>
   (function($) {
     $(document).ready(function() {
-      $('video,audio').mediaelementplayer({
+
+      var media_items = $('video').filter(function() {
+        return $(this).parent().prop("tagName") !== "PICTURE";
```

Comment:
tagName is no method of jquery and parent returns an jQuery element. So this only works when we use plain javascript but jquery is required so why we shouldn`t use it? Even the filter method is jQuery too :)

See this jsbin:
https://jsbin.com/toxicemoqi/edit?html,js,console,output

Prop returns a string and we could lowercase the value if you like.